pkg/cmd/tests/options.go

```diff
@@ -63,6 +64,7 @@ func NewTestFrameworkOptions(streams genericclioptions.IOStreams, userAgent stri
 			NodeServiceType:             string(scyllav1.NodeServiceTypeHeadless),
 			NodesBroadcastAddressType:   string(scyllav1.BroadcastAddressTypePodIP),
 			ClientsBroadcastAddressType: string(scyllav1.BroadcastAddressTypePodIP),
+			StorageClassName:            "scylladb-local-xfs",
```

Comment:
nit: I think empty is a good default given how default storageclasses work, I don't think this binary is the place to be opinionated about naming. We'd always set the flag explicitly in our tests thought.

Reply:
> I think empty is a good default given how default storageclasses work

can you elaborate? I don't understand why that'd make it a good default. Empty storageClassName means you won't get any storageClass, it's not equivalent to not setting it at all.